README.md

```diff
@@ -24,3 +24,27 @@ We are using PostgreSQL to store information. You need to have two databases loc
 ### Using Slack App in Development
 
 We use `ngrok` to route traffic from the internet to our machine. See guide: https://api.slack.com/tutorials/tunneling-with-ngrok
+
+### Guide to use the Lunchinator App
+
+To use the app, there are a list of commands which need to be run on the lunchinator app by the admin.
+
+- A `LunchCycle` refers to a cycle of 6 weeks (this may increase or decrease depending on number of MT employees). A LunchCycle has the capacity to allow each staff member to join a friday lunch at least once. Each week of the lunch cycle contains the restaurant selected, the lunchers who have been drawn, and the date.
+
+- A LunchCycle should be created a week before the previous LunchCycle ends. To do this, run the command `/lunchinator_new`. This will show you a preview message which will be sent out to all slack users.
+
+- If you would like to alter the selection of restaurants previewed. See sheet: https://docs.google.com/spreadsheets/d/1uvAn-YIWggsbigWLVQJcukRxyR85rFYd0igJLvrHXes/edit#gid=0
+
+- Rerun the command to create a new LunchCycle if the retsurant list has been altered.
+
+- Once you are happy with the preview message, send the messages to all slack users using the `/lunchinator_availabilities` command. The slack users should respond with their availabilities.
+
+- Slack users will have a week to respond to the message. You can send reminders to people who are yet to submit their reactions using the `/lunchinator_send_reminder` command.
```

Comment:
so they don't send automatically every half day until the draw is done?

Reply:
Not at this stage - the way the scheduler works on heroku it would require someone to manually turn the scheduler on when the new lunch cycle is created and then off once the draw is done

Reply:
is there a way to track how many responses we're missing? to compare with the number of people on holiday, and know if we need to keep chasing or if we can do the draw?

Reply:
Yeah, there is a switch on the UI which will display all the non responders.